observability/micrometer/src/main/java/org/wildfly/extension/micrometer/MicrometerSubsystemModel.java

```diff
@@ -9,9 +9,10 @@
 
 public enum MicrometerSubsystemModel implements SubsystemModel {
     VERSION_1_0_0(1, 0, 0),
-    VERSION_1_1_0(1, 1, 0);
+    VERSION_1_1_0(1, 1, 0),
+    VERSION_1_1_0_COMMUNITY(1, 1, 0);
```

Comment:
This should be `VERSION_2_0_0_COMMUNITY(2, 0, 0)` and we need a `VERSION_2_0_0(2, 0, 0)` as well. Basically what you did for `MicrometerSubsystemSchema`.

Reply:
@jasondlee See https://wildfly.zulipchat.com/#narrow/channel/174184-wildfly-developers/topic/WFLY-18582.20Prometheus.20Endpoint.20to.20Micrometer for some details. Maybe the community version is not needed because the attributes themselves are set to a community stability level.

Reply:
Yup. Just missed this. Changes incoming...

Reply:
Fixed